src/mame/drivers/segas24.cpp

```diff
@@ -1182,28 +1201,65 @@ void segas24_state::decrypted_opcodes_map(address_map &map)
 
 void segas24_state::machine_start()
 {
+	if ((m_romboard != nullptr) && (m_rombank[0] != nullptr) && (m_rombank[1] != nullptr))
+	{
+		uint8_t *usr1 = m_romboard->base();
+		m_rombank[0]->configure_entries(0, 16, usr1, 0x40000);
+		m_rombank[1]->configure_entries(0, 16, usr1, 0x40000);
+
+		save_item(NAME(m_curbank));
+	}
+
 	if (m_track_size)
+	{
 		subdevice<nvram_device>("floppy_nvram")->set_base(&m_floppy[0], 2*m_track_size);
 
-	if (m_romboard != nullptr)
-	{
-		uint8_t *usr1 = m_romboard->base();
-		membank("bank1")->configure_entries(0, 16, usr1, 0x40000);
-		membank("bank2")->configure_entries(0, 16, usr1, 0x40000);
+		machine().save().register_postload(save_prepost_delegate(FUNC(segas24_state::postload), this));
```

Comment:
The `register_postload` API is deprecated now. Please use `device_post_load` instead.